js/views/stock.js

```diff
@@ -1 +1,64 @@
 // define your StockView here
+
+
+// - `StockView`
+//   - `model`: points to an instance of a Stock model.
+//   - `render`: updates the HTML on the page to reflect the info of the associated stock model. The info is the price, companyName and totalValue.
+//   - 'lookupSymbol': when the user enters a symbol, this should instantiate a new instance of a stock model and associate it with the `model` attribute of the view. It should then re-`render` the view.
+//   - updateShares: when the user enters a number of shares, it should update the existing associated stock model, and then re-render the view
+
+
+var StockView = function(){
+	this.symbolButton = document.getElementById("new_stock_button");
+	this.symbolInput = document.getElementById("new_stock_text");
+	this.symbolButton = addEventListener("click", this.createStock.bind(this));
+	this.sharesButton = document.getElementById("shares_updater_button");
+	this.sharesInput = document.getElementById("shares_updater_text");
+	this.stocks = document.getElementId("stocks")
+}
+
+StockView.prototype = {
+	createStock: function(event){
+		this.stock = new Stock(this.symbolInput.value);
+		console.log(this.stock);
+		this.render;
+	}
```

Comment:
cool, I like this portion, and this is kind of what i was talking about in your `application.js`